src/javascript/JContent/actions/openInLiveAction/openInLiveAction.js

```diff
@@ -22,7 +22,10 @@ export const OpenInLiveActionComponent = ({
         skip: !path
     });
 
-    if (loading || error || !data || !data.jcr.result.publicationInfo.existsInLive || data.jcr.result.publicationInfo.status === 'NOT_PUBLISHED' || data.jcr.result.publicationInfo.status === 'UNPUBLISHED') {
+    if (loading || error || !data || !data.jcr.result.publicationInfo.existsInLive ||
+        data.jcr.result.publicationInfo.status === 'NOT_PUBLISHED' ||
+        data.jcr.result.publicationInfo.status === 'UNPUBLISHED' ||
+        (!data.jcr.result.previewAvailable && data.jcr.result.displayableNode === null)) {
```

Comment:
Should we also check undefined (or falsely)

Reply:
no, coming from graphql, values are null, not undefined